skema/program_analysis/comment_extractor/tests/examples/comments.c

```diff
@@ -0,0 +1,19 @@
+// Single line comment 
+
+/* Multi line comment on single line*/
+
+/*
+    Multi line comment 
+    on multiple lines
+*/
+
+/* Docstring in function declaration*/
+int foo();
+
+int foo(){
+	/*
+        Docstring in function definition
+    */
```

Comment:
Does the spacing on Line 16 matter here for the docstring? Does it need to match the spacing on line 14 or no? 
Ditto on the c++ example.

Reply:
The tree-sitter node structure is the same either way, so it wouldn't matter for extraction. However, I'll correct it just so the spacing is consistent.